website/content/docs/concepts/secure-variables.mdx

```diff
@@ -0,0 +1,124 @@
+at all paths in the "dev" namespace that are prefixed with "project/", but only
+read access to paths prefixed with "system/". Note that the glob can match an
+empty string but all other characters are strictly matched. This policy grants
+read access to paths prefixed with "system/" but not a path named "system"
+(without a trailing slash).
+
+```hcl
+namespace "dev" {
+  policy       = "write"
+  capabilities = ["alloc-node-exec"]
+
+  secure_variables {
+
+    # full access to secrets in all "project" paths
+    path "project/*" {
+      capabilities = ["write", "read", "destroy", "list"]
+    }
+
+    # read/list access within a "system/" path belonging to administrators
+    path "system/*" {
+      capabilities = ["read"]
+    }
+  }
+}
+```
+
+The available capabilities for Secure Variables are as follows:
+
+| Capability | Notes                                                                                                                        |
+|------------|------------------------------------------------------------------------------------------------------------------------------|
+| write      | Create or update Secure Variables at this path. Includes the "list" capability but not the "read" or "destroy" capabilities. |
+| read       | Read the decrypted contents of Secure Variables at this path. Also includes the "list" capability                            |
+| list       | List the metadata but not contents of Secure Variables at this path.                                                         |
+| destroy    | Delete Secure Variables at this path.                                                                                        |
+
+## Task Access to Secure Variables
+
+In Nomad 1.4.0 tasks can access Secure Variables only with the [`template`]
+block. The [workload identity] for each task grants it automatic access to
+Secure Variables found at Nomad-owned paths with the prefix `nomad/jobs/`,
+followed by the job ID, task group name, and task name.
+
+```
+nomad/jobs/$job_id/$task_group/$task_name
+nomad/jobs/$job_id/$task_group
+nomad/jobs/$job_id
+nomad/jobs
+```
+
+For example, the task created by the example job spec created with `nomad job
+init -short` will automatically have access to Secure Variables at the following
+paths, within its own `default` namespace:
+
+```
+nomad/jobs/example/cache/redis
+nomad/jobs/example/cache
+nomad/jobs/example
+nomad/jobs
+```
+
+You can provide access to additional secrets by creating policies associated
+with the task's workload identity. For example, to give the task above access to
+set of shared secrets, you can create the following policy file:
+
+```hcl
+namespace "shared" {
+  secure_variables {
+    path "*" {
+      capabilities = ["read"]
+    }
+  }
+}
+```
+
+Then create the policy with `nomad acl policy apply
+"_:/default/example/cache/redis" ./policy.hcl` to give the task read access to
+all paths in the "shared" namespace.
```

Comment:
would this look cleaner with full code blocks rather than inline snippets?

Reply:
Done